### DISCUSSION TITLE
How to use fillment to achieve similar shadow effects

### DISCUSSION THREAD

**Discussion by fuchao** [google/filament#7443] [Q&A] [answered]
![微信图片_20231219224440](https://github.com/google/filament/assets/1783838/bc4c2171-5969-4419-b79b-96fff320000a)

**Comment by romainguy** [comment, marked as answer]
This looks like a pre-baked shadow drawn as a textured quad under the car. For the darkening seen under the top car, SSAO is likely what you want.

**Reply by fuchao** [reply]
Thank you, it seems that the effect is quite similar at the moment.
